src/nginxconfig/templates/domain_sections/php.vue

```diff
@@ -65,6 +65,53 @@ THE SOFTWARE.
             </div>
         </div>
 
+        <div v-if="php" class="field is-horizontal">
+            <div class="field-label has-margin-top">
+                <label class="label">{{ $t('templates.domainSections.php.phpServer') }}</label>
+            </div>
+            <div class="field-body">
+                <div class="field">
+                    <div :class="`control${phpServerChanged ? ' is-changed' : ''}`">
+                        <VueSelect v-model="phpServer"
```

Comment:
```suggestion
                        <VueSelect ref="phpServerSelect"
                                   v-model="phpServer"
```
Looks like this is missing a ref needed for the translation watcher